php/whatsapp-with-vonage/src/index.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+require(__DIR__ . '/../vendor/autoload.php');
+require(__DIR__ . '/utils.php');
+
+
+use Firebase\JWT\JWT;
+use Firebase\JWT\Key;
```

Comment:
This is never used in the function and can be removed, as well as the composer package `firebase/php-jwt`.

Reply:
Thanks for pointing out :)